scripts/database_delete.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+docker stop perfectweek-db || true && docker rm perfectweek-db || true
```

Comment:
Why the `|| true &&` ? Use `;`

```sh
docker stop perfectweek-db ; docker rm perfectweek-db
```

Furthermore, you should use `-f` to remove a running container

```sh
docker rm -f perfectweek-db
```